rtv/utils.py

```diff
@@ -118,7 +118,7 @@ def load_config():
 
     return defaults
 
-def text_input(window):
+def text_input(window, show_cursor=False, insert_mode=True):
```

Comment:
Yes. I think it is another `curses` bug (similar to the one described [here](http://bugs.python.org/issue13051))?

There are other issues of `curses.textpad`:
1. Infinite recursion bug when `insert_mode=True`. I actually went into the `curses` source and fixed it.
2. `stripspaces` flag. It it is true, it strips all the unnecessary spaces and line-changes, effectively destroying Reddits' markdown. If it false, every single space in textbox is returned as a part of string.

I think we have to implement a mini text editor in place.